datahub-web-react/src/app/entity/shared/containers/profile/utils.ts

```diff
@@ -202,3 +202,22 @@ export function getOnboardingStepIdsForEntityType(entityType: EntityType): strin
             return [];
     }
 }
+
+function sortTabsWithDefaultTabId(tabs: EntityTab[], defaultTabId: string) {
+    return tabs.sort((tabA, tabB) => {
+        if (tabA.id === defaultTabId) return -1;
+        if (tabB.id === defaultTabId) return 1;
+        return 0;
+    });
```

Comment:
Not sure if this could lead to shuffling depending on implementation of sort. It's probably stable if merge sort, but to avoid you could just splice the default tab out of the list and put it in the front. Not blocking if this works though.

Reply:
`const sortedTabs = tabs.filter(t => t.id === defaultTabId).concat(tabs.filter(t => t.id !== defaultTabId))`

Reply:
Let's definitely be careful about this one. Hoping that in existing cases we do not change the default sort order

Reply:
yeah this does indeed work as is and I believe returning `1` `-1` or `0` is pretty standard for the classic javascript `sort`. I think i'll leave as is. also double checked this does not affect sorting for other entity types and we go back to default sorting on domains if the env variable isn't set for what we're looking for